spec/features/admin_invoices/index_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe 'the admin invoice index' do
+  describe 'as an admin' do
+    before :each do
+      customer = Customer.create!(first_name: 'Taylor', last_name: 'Swift')
+      @invoice_1 = customer.invoices.create!(status: 'in progress')
+      @invoice_2 = customer.invoices.create(status: 'cancelled')
+      @invoice_3 = customer.invoices.create(status: 'completed')
+    end
```

Comment:
Should these be @customer if they are used within a before :each block?

Reply:
customer isn't ever being referenced in the tests which is why I didn't make it an instance variable. not sure if that's important for performance or anything but I just assumed the less instance variables, the better 🤷‍♂️